platform-tests/src/test/java/org/junit/platform/reporting/open/xml/OpenTestReportGeneratingListenerTests.java

```diff
@@ -191,7 +191,7 @@ void includesGitInfoWhenEnabled(String originUrl, @TempDir Path tempDirectory) t
 
 		execGit(tempDirectory, "config", "user.name", "Alice");
 		execGit(tempDirectory, "config", "user.email", "alice@example.org");
-		execGit(tempDirectory, "commit", "-m", "Initial commit");
```

Comment:
This will work, but maybe creating an empty config file in the tmp dir and setting `GIT_CONFIG_GLOBAL` would be more durable.

https://git-scm.com/docs/git#Documentation/git.txt-GITCONFIGGLOBAL